superset-frontend/src/addSlice/AddSliceContainer.tsx

```diff
@@ -195,10 +198,12 @@ export default class AddSliceContainer extends React.PureComponent<
   }
 
   exploreUrl() {
+    const dashboardId = getUrlParam(URL_PARAMS.dashboardId);
     const formData = encodeURIComponent(
       JSON.stringify({
         viz_type: this.state.visType,
         datasource: this.state.datasource?.value,
+        ...(!isNullish(dashboardId) && { dashboardId }),
```

Comment:
this is a cool function, why nullish?

Reply:
`!dashboardId` would exclude `dashboardId === 0`, which is a valid id 😄